### PR TITLE
Added support to get the most recent reviews of an app

### PR DESCRIPTION
**Pull request by dhruvpeter**
A new field `reviews_order` added to "config.yaml" to specify most recent or most relevant reviews to be crawled.
If it's value is `"most recent"` , the most recent reviews are crawled.
If it's value is `"most relevant"` , the most relevant reviews are crawled.
If no value is given in the field, most relevant reviews are crawled.
#1

### CODE REVIEW

**Comment by abose** [comment]
Thanks for contributing @dhruvpeter 
[Can you run the code through pep8 linter](https://code.visualstudio.com/docs/python/linting)

File: src/crawler/scraper.py [hunk 1]
```diff
@@ -75,13 +77,19 @@ def __print_summary(self):
         table = tabulate(results_to_print, ['Score', 'Number Of Entries'])
         print(table)
 
+    def __sort_reviews(self):
+        if Reviews_order(self.reviews_order).name == 'MOST_RECENT':
+            return Sort.NEWEST      #get most recent reviews
+        elif Reviews_order(self.reviews_order).name == 'MOST_RELEVANT' :
+            return Sort.MOST_RELEVANT       #get most relevant reviews
```
Comment (charlypa) [inline src/crawler/scraper.py]: For clarity use Reviews_order(self.reviews_order) == Reviews_order.MOST_RECENT

Comment (charlypa) [reply src/crawler/scraper.py]: Have you verified that by changing the sort order we will get different results.

Comment (dhruvpeter) [reply src/crawler/scraper.py]: > If no sort order is specified take Sort.MOST_RELEVANT as default value

If reviews_order has no value specified, then value will be assigned to "most relevant" . This is done in line 138 of main.py

Comment (dhruvpeter) [reply src/crawler/scraper.py]: > Have you verified that by changing the sort order we will get different results.

Yes, I have verified and got different results for most recent and most relevant sort orders

Comment (charlypa) [reply src/crawler/scraper.py]: Looks good for me

**Review by charlypa — APPROVED** [review]

**Comment by charlypa** [comment]
@dhruvpeter  good work. You may merge the code to main

**Pull request merged**